recipes/yagmail/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "yagmail" %}
+{% set version = "0.11.224" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ca5ff7eee3c190eedd7cbdf05bff69c5c0ae357aef64f89dcc42e4872c6ebe7c
+
+build:
+  number: 0
```

Comment:
Can this be `noarch: python` ? See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

Reply:
Nice, wasn't aware of that option. Just added it. Thanks, @isuruf!